source/Crew.cpp

```diff
@@ -297,6 +297,34 @@ int64_t Crew::ShareProfit(const std::vector<std::shared_ptr<Ship>> &ships, const
 
 
 
+double Crew::CalculateProfitShare(const std::vector<std::shared_ptr<Ship>> &ships, const Ship * flagship)
```

Comment:
Cool. Could you please delete `Crew::ShareProfit` as part of this change, and update any places where it's called to use `Crew::CalculateProfitShare` instead?

That way, we won't have the exact same logic duplicated in two places.